### PR TITLE
AC Automated Test - LRAC-8323 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-8323

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 1]
```diff
@@ -182,12 +182,75 @@ definition {
...
+
+			ACSettings.goToProperties();
+		}
 
-		// TODO LRAC-8323
+		task ("Checks if only property 2 and property 3 are present") {
+			ACProperties.assertPropertyTitleList(propertyName = "Property Name 1");
+		}
+
+		task ("Log in as the user corbin.murakami@faro.io, go to the properties menu and delete all properties") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:221]: hi @rafaellajordao , could you please update the task description?

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase]: fixed @alexcandidos !! sorry about this!!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Review by alexcandidos — COMMENTED** [review]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**